client/starwhale/core/dataset/copy.py

```diff
@@ -141,3 +149,18 @@ def _do_download_bundle_dir(self, progress: Progress) -> None:
                 local._info = copy.deepcopy(remote.info)
 
         super()._do_download_bundle_dir(progress)
+
+    def _get_remote_project_name(self, instance: URI, project: str) -> Any:
+        resp = self.do_http_request(
+            f"/project/{project}",
+            instance_uri=instance,
+            method=HTTPMethod.GET,
+            use_raise=True,
```

Comment:
if `use_raise=True`, the http request will check the status code automatically, so can we ignore line160-161?

Reply:
done